code/datums/black_market_item.dm

```diff
@@ -240,6 +240,21 @@ for anyone but the person committing mass murder.
 /datum/black_market_item/tech
 	category = "Advanced Technology" 
 
+/datum/black_market_item/tech/emagged_nikita
+	name = "Emagged Nikita"
+	desc = "A rocket launcher that fires missiles that you can ride. Great for special effects in Live-Action movies and terrorist roleplay."
+	item = /obj/structure/closet/crate/secure/weapon/experimental/nikita
+	sps_chances = list(0, 20, 35)
+	stock_min = 1
+	stock_max = 1
+	cost_min = 1000
+	cost_max = 2000
```

Comment:
That costs too much, how am I, a lowly roleplaying assistant, meant to roleplay with such high costs?

Reply:
no worries, fren. got u covered #23588